init.h

```diff
@@ -68,6 +68,7 @@
 #define R_SIDEBAR       (1<<7)  /* redraw the sidebar */
 #define R_BOTH		(R_INDEX | R_PAGER)
 #define R_RESORT_BOTH	(R_RESORT | R_RESORT_SUB)
+#define R_SENSITIVE     (1<<8)  /* hide in mutt -C */
```

Comment:
cf earlier comments, choose a prefix to properly namespace this, like `Q_` for qualifier…